controllers/front/OrderConfirmationController.php

```diff
@@ -97,6 +97,7 @@ public function initContent()
             'HOOK_PAYMENT_RETURN' => $this->displayPaymentReturn($order),
             'order' => $presentedOrder,
             'register_form' => $register_form,
+            'registered_customer_exists' => Customer::customerExists($this->context->customer->email, false, true),
```

Comment:
```suggestion
            'registered_customer_exists' => Customer::customerExists($this->context->customer->email, false, true),
```

What if it's a guest? the customer will be empty no?

Reply:
@PierreRambaud The last true parameters says, that it should search for guests only. Customer exists -> returns true, Nobody or guests exist -> returns false.

Reply:
I think about `$this->context->customer->email` the `$this->context->customer` can be null

Reply:
@PierreRambaud Original code is using the variable two lines down, it's removed there. Cannot be empty, in normal case...